client/src/context/interface/Profile.ts

```diff
@@ -0,0 +1,31 @@
+import { User } from './User';
+
+export interface Profile {

```

Comment:
There is an interfaces folder in the `src/` directory, there is no need to create a new directory for interfaces.

Reply:
Oops! Thank you.